src/navigator/TabNavigator.test.tsx

```diff
@@ -0,0 +1,83 @@
+import { CommonActions } from '@react-navigation/native'
+import { fireEvent, render } from '@testing-library/react-native'
+import React from 'react'
+import { Provider } from 'react-redux'
+import { Screens } from 'src/navigator/Screens'
+import TabNavigator from 'src/navigator/TabNavigator'
+import { getFeatureGate } from 'src/statsig'
+import { StatsigFeatureGates } from 'src/statsig/types'
+import MockedNavigator from 'test/MockedNavigator'
+import { createMockStore } from 'test/utils'
+
+jest.mock('src/statsig')
+
+describe('TabNavigator', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+    jest
+      .mocked(getFeatureGate)
+      .mockImplementation((featureGate) => featureGate !== StatsigFeatureGates.USE_TAB_NAVIGATOR)
```

Comment:
personally not a fan of this pattern as it sets every other gate to true, but I guess this is good enough for documenting what gate we really care about in these tests.

Reply:
I am open to other suggestions. It looks like mockImplementation always expects a boolean value returned so it's a bit tricky in that regard.